### PR TITLE
add transport configure file for paratera-th2

### PR DESCRIPTION
**Pull request by unkcpz**
As point out in issue #11. transport configuration file also include if
there are particular parameters specifed.

### CODE REVIEW

File: computers/paratera-th2/paratera-th2-configure.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+port:   2200
+look_for_keys:          True
+key_filename:           '/home/unkcpz/.ssh/papp-test.id'
```
Comment (ltalirz) [inline computers/paratera-th2/paratera-th2-configure.yml:4]: I guess these two settings should be configured by the users themselves?

File: computers/paratera-th2/paratera-th2-setup.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (ltalirz) [inline computers/paratera-th2/paratera-th2-setup.yml:1]: could you please delete the corresponding file without -setup?

Comment (ltalirz) [reply computers/paratera-th2/paratera-th2-setup.yml:1]: if we're moving to the `-setup` and `-configure` nomenclature now, please also rename the files in the other subfolders and adapt the README.md.

Thanks a lot!

File: computers/paratera-th2/paratera-th2-configure.yml (hunk 1 above)
Comment (unkcpz) [reply computers/paratera-th2/paratera-th2-configure.yml]: One question here, `look_for_keys` must be set to `True` if `key_filename` is specified here and the filename is not the default `id_rsa`, correct?

File: computers/paratera-th2/paratera-th2-setup.yml (hunk 2 above)
Comment (unkcpz) [reply computers/paratera-th2/paratera-th2-setup.yml]: Sorry, my fault. A little fading out after taking cold pills :p 
 fixed.

File: computers/paratera-th2/paratera-th2-configure.yml (hunk 1 above)
Comment (ltalirz) [reply computers/paratera-th2/paratera-th2-configure.yml]: From to the [docs](https://aiida.readthedocs.io/projects/aiida-core/en/latest/_modules/aiida/transports/plugins/ssh.html?highlight=look_for_keys), I understand it is needed only when you don't specify the path to the key file.
In any case, it is true by default.

**Comment by unkcpz** [comment]
Thanks! I'll create new pr to rename computer files.

**Pull request merged**